assets/scss/local.scss

```diff
@@ -4,4 +4,10 @@
 
 .govuk-panel-confirmation-error {
   background: darkred;
-}
\ No newline at end of file
+}
+
+.single-row-textarea {
+  resize: none!important;
```

Comment:
Are these `!important` actually needed? I'd really rather do without them

Reply:
looks like i don't need `!important` indeed
i have them in csip-ui too, we might try remove them over there as well